hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/tools/federation/RouterAdmin.java

```diff
@@ -140,7 +143,7 @@ private void printUsage(String cmd) {
   private String getUsage(String cmd) {
     if (cmd == null) {
       String[] commands =
-          {"-add", "-update", "-rm", "-ls", "-getDestination", "-setQuota",
+          {"-add", "-addall", "-update", "-rm", "-ls", "-getDestination", "-setQuota",
```

Comment:
Why don’t we adjust it in add only? Why we need a new command?

just make add take multiple entries rather than 1? If there are more than one entries, your code, else the present one

Reply:
I think it sort of breaks compatibility for `-add` no? also, by keeping separate command like `-addall`, we are able to clearly express that we can add more mount points with it, so it is better and clean separation?

Reply:
Nopes, compat won't break, adding extra options is allowed, Unless existing add command works as is, like doesn't change in behaviour, we are good from compat point of view.

Even **mkdir** behaves in similar way
```
bash-4.2$ hdfs dfs -mkdir /dir /dir1 /dir2 /dir3 /dir4
bash-4.2$ hdfs dfs -ls /
Found 5 items
drwxr-xr-x   - hadoop supergroup          0 2023-04-13 19:08 /dir
drwxr-xr-x   - hadoop supergroup          0 2023-04-13 19:08 /dir1
drwxr-xr-x   - hadoop supergroup          0 2023-04-13 19:08 /dir2
drwxr-xr-x   - hadoop supergroup          0 2023-04-13 19:08 /dir3
drwxr-xr-x   - hadoop supergroup          0 2023-04-13 19:08 /dir4
```

It can take multiple entries, so we can do for -add as well. Juzz my thoughts, if @goiri is good with a new command, consider no objections from me as well :)